src/test/java/io/jenkins/plugins/analysis/core/charts/HealthTrendChartTest.java

```diff
@@ -0,0 +1,69 @@
+package io.jenkins.plugins.analysis.core.charts;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import edu.hm.hafner.analysis.Severity;
+
+import io.jenkins.plugins.analysis.core.model.AnalysisResult.BuildProperties;
+import io.jenkins.plugins.analysis.core.util.AnalysisBuild;
+import io.jenkins.plugins.analysis.core.util.AnalysisBuildResult;
+import io.jenkins.plugins.analysis.core.util.HealthDescriptor;
+
+import static java.util.Arrays.*;
+import static net.javacrumbs.jsonunit.assertj.JsonAssertions.*;
+import static org.mockito.Mockito.*;
+
+
+/**
+ * Tests the class {@link HealthTrendChart}.
+ *
+ * @author Matthias Herpers
+ */
+
+class HealthTrendChartTest {
+
+    /**
+     * Verifies that a HealthTrendChart with two Results shows right x/Axis labels.
+     */
+    @Test
+    void shouldCreateHealthChartForJobAndMultipleActions() {
+        int healty = 5;
+        int unhelthy = 10;
+        Severity minSeverity = new Severity("TestSeverity");
+        HealthDescriptor healthDescriptor = new HealthDescriptor(healty, unhelthy, minSeverity);
```

Comment:
If you don't use healthy and unhealthy more than once then there is no need to create variables